### DISCUSSION TITLE
How to change the layout being used?

### DISCUSSION THREAD

**Discussion by cyberbrix** [SubtitleEdit/subtitleedit#7825] [Q&A]
This should be intuitive, but every time I try to use the layout button, I get a warning about being in un-docked mode. I cannot find any controls or options for what this means.

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/SubtitleEdit/subtitleedit/assets/75588964/089d6490-bab5-46cd-8c8b-2b9746742b3e)

You'll want to click that so it opens in a new browser window, where you can magnify it & actually read what look like chicken scratches here.

**Comment by cyberbrix** [comment]
Under video, mine just says "un-dock video controls" and when I click it, nothing changes or happens

**Comment by GrampaWildWilly** [comment]
I think we're rapidly reaching the limit of my expertise.  And I'm probably grossly flattering myself by calling it expertise.  But that's always the point where I accept the challenge of simply winging it.  I must say up front that I don't really know what I'm talking about here so if you make any manual changes to Settings.XML, make a safe copy of it first.

I found this block of lines in my Settings.XML file:
```
    <Undocked>True</Undocked>
    <UndockedVideoPosition>1911;-9;1938;1098</UndockedVideoPosition>
    <UndockedVideoFullscreen>False</UndockedVideoFullscreen>
    <UndockedWaveformPosition>1120;538;763;248</UndockedWaveformPosition>
    <UndockedVideoControlsPosition>4;644;239;340</UndockedVideoControlsPosition>
```
I'm going to take a wild guess & say your Undocked tag says True.  I would then manually change it to False.  But do that while SE is not running.  See if that does anything.  I think if that tag has the False value, the other 4 tags don't mean anything.  I would assume that if you subsequently were to undock your controls, those tags would get new values.

If that doesn't work, I would just remove those 5 tags, the entirety of all 5 lines, again with SE not running, then see what happens next time you open SE.  My hope is it would regenerate whatever it needs there.

And make sure your backup copy of Settings.XML is in a safe place in case this advice is simply scatterbrained.  In which case you would just copy it back into place & we're back at square 0 waiting for somebody who actually knows what they're talking about to join the discussion.

**Comment by cyberbrix** [comment]
Exactly right! It was the undocked setting in the XML file. Changed it to False and it worked.

**Comment by GrampaWildWilly** [comment]
Sometimes even a blind squirrel finds an acorn in the forest.

This gives me 2 thoughts.  How did that tag in Settings.XML get out of synch with the
GUI?  What do those lines in your Settings.XML look like now?  I'm sure neither of us can
answer that first question.  You can answer the second.

**Reply by cyberbrix** [reply]
<Undocked>False</Undocked>
    <UndockedVideoPosition>-32000;-32000</UndockedVideoPosition>
    <UndockedVideoFullscreen>False</UndockedVideoFullscreen>
    <UndockedWaveformPosition>-32000;-32000</UndockedWaveformPosition>
    <UndockedVideoControlsPosition>-32000;-32000</UndockedVideoControl
